Source/WebCore/inspector/agents/InspectorDOMAgent.cpp

```diff
@@ -329,10 +334,13 @@ void InspectorDOMAgent::willDestroyFrontendAndBackend(Inspector::DisconnectReaso
     m_inspectedNode = nullptr;
 
     Protocol::ErrorString ignored;
-    setSearchingForNode(ignored, false, nullptr, false);
+    setSearchingForNode(ignored, false, nullptr, nullptr, nullptr, false);
```

Comment:
Aside: this is an unfortunately opaque set of parameters 😅

Reply:
luckily it's in the same file, so looking at the definition is pretty easy

but yeah would be more ideal to have some sort of clarity in there (especially in a way that conforms to WebKit style)